site/content/entry/yuzu-progress-report-may-2022/index.md

```diff
@@ -0,0 +1,235 @@
+
+However in practice, most modern operating systems (Windows/Linux) use hybrid mutexes and hybrid spinlocks.
+So spinlocks are fine on systems with tons of cores, as it's not a big deal for host threads to spin. 
+On low core counts, you really start starving the system by doing that, whereas a mutex will allow another thread to be scheduled on that core/thread.
+
+Thus {{< gh-hovercard "8172" "by moving from spinlocks to mutexes," >}} we were able to improve how yuzu runs on systems with low core counts. 
+Our testing results showed that yuzu is now much more usable on 4 thread systems.
+
+## UI changes
+
+[Docteh](https://github.com/Docteh) has been very helpful with some translation holes we had for a while.
+
+For example, the Custom RTC setting had several issues if the Windows system locale was set in certain languages, making it either display incorrectly (for example lacking the AM/PM indicator), or completely unusable.
+{{< gh-hovercard "8291" "Fixing the display format" >}} allows Custom RTC to show up correctly in any language now.
+
+The Network tab in `Emulation > Configure… > System` could remain untranslated after changing languages. This was a simple case of forgetting to include the tab in the translations, so Docteh {{< gh-hovercard "8293" "fixed the oopsie" >}} and the lone Network tab now displays as it should.
+
+{{< single-title-imgs-compare
+    "Netto-kun. Now I want a Battle Network Legacy Collection, c'mon CAPCOM!"
+    "./netbug.png"
+    "./netfix.png"
+>}}
+
+yuzu’s icon in the About dialog has been in a struggle lately, a fix for Linux builds would break Windows support, and vice versa.
+Via qtcreator, Docteh {{< gh-hovercard "8339" "fixed the About dialog UI file," >}} and removed an old warning caused by the original .png image.
+Thanks for taking the time to properly address the issue once and for all!
+
+## Controller changes
+
+Motion continued reporting data when disabled, causing `Pokémon Let’s Go, Eevee/Pikachu!` to spam `StopSixAxisSensor` errors in the log.
+While working on this, [german77](https://github.com/german77) noticed a missing parameter, `delta_time`.
+Its implementation allows yuzu to have an {{< gh-hovercard "8308" "accurate motion refresh rate," >}} equal to the Switch.
+
+In an all-in-one pull request, german77 made {{< gh-hovercard "8368" "several input changes," >}} including:
+
+- Add proper error handling for several [HID](https://en.wikipedia.org/wiki/Human_interface_device) functions
+- Improve previous implementations to match more closely to native hardware.
+- Implement functions needed by `Nintendo Switch Sports`, `EnableSixAxisSensorUnalteredPassthrough`, `IsSixAxisSensorUnalteredPassthroughEnabled`, `LoadSixAxisSensorCalibrationParameter`, `GetSixAxisSensorIcInformation`, `ResetIsSixAxisSensorDeviceNewlyAssigned`.
+
+Nintendo Switch Sports will only be playable after an audio rewrite and more GPU work.
```

Comment:
Replace with:

"While we've made some great progress here, Nintendo Switch Sports remains unplayable on yuzu until we rework our audio implementation, and furthermore make some much needed GPU fixes. While audio and perfect rendering may not seem critical to playability, games often are quite unstable if these are not accurate. Rest assured, we're working on these and will have more to share very soon!"